CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ the code was deployed.
 
 ### Fixes
```

Comment:
I see that this wasn't new in this PR, but could you please change this to "Fixed" to conform with the Keep A Changelog categories (https://keepachangelog.com/en/1.0.0/)?